Fixing issue and improved network performance and added loader etc..

rjoydip opened the pull request:
This PR contains below points.

***Description***

- fix(warning): Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function 
- fix(styled-component): To resolve this only create new Styled Components outside of any render method and function component 
- improved(rendering): Very slow network rendering (almende/vis#3187)
- fix(state/travel): Error: Array index '-1' has to be an integer
- fix(travel:filter): TypeError: Cannot read property 'forEach' of undefined
- feat: loader functionality added
- network cluster performance improved
- memory leaking issues fixed some of the cases 

Note: Loader designer is very much basic. Fell free to improve `loader` design after merged the PR.

sibeshkar commented:
Thanks for the fixes. the extra squigliness in the edges seems to be destroying any performance optimization you made. Possible to fix? Also needs a better loader, that can be located at the top left. Thanks :)

rjoydip commented:
@sibeshkar 
> the extra squigliness in the edges seems to be destroying any performance optimization you made
Fixed. But changed all png logos to svg for better rendering. Also changed others logo

- Images are taken from [flaticon](https://www.flaticon.com/)
- Changed those gradient colored images if required

***Note: For further performance improvement and better resource loading you have to merge #171 PR**
